src/python/grpcio/grpc/__init__.py

```diff
@@ -1252,6 +1269,65 @@ def ssl_server_credentials(private_key_certificate_chain_pairs,
             ], require_client_auth))
 
 
+def ssl_server_credentials_with_cert_config_fetcher(cert_config_fetcher_cb,
+                                                    require_client_auth=False):
+    """Creates a ServerCredentials for use with an SSL-enabled Server.
+
+    Args:
+      cert_config_fetcher_cb (callable): a callback that takes no
+        arguments and should return a ServerCertificateConfig to
+        replace the server's current cert, or None for no change
+        (i.e., the server will continue its current certificate
+        config). The library will call this callback once during
+        server instantiation to obtain the initial certificate
+        config---a valid ServerCertificateConfig must be returned in
+        this case. Then, the library will call this callback on
+        *every* new client connection before starting the TLS
+        handshake with the client, thus allowing the user application
+        to optionally return a new ServerCertificateConfig that the
+        server will then use for the handshake.
+      require_client_auth: A boolean indicating whether or not to
+        require clients to be authenticated. May only be True if
+        root_certificates is not None.
+
+    Returns:
+      A ServerCredentials for use with an SSL-enabled
```

Comment:
Just "A ServerCredentials."? The returned value should be valid for use anywhere that a ServerCredentials is valid for use. No need to call attention at this code location to where those other locations might be.

Reply:
just following the lead of `def ssl_server_credentials`, but i should remove the second sentence then?

Reply:
done: just `A ServerCredentials.` https://github.com/grpc/grpc/pull/13188/commits/67ed381d0a8905537e7b31908336cfe8543838b5